app/icons/ChevronLast.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { forwardRef } from 'react'
```

Comment:
Similar as previous PRs: move this to `/lib`, it'll be used by multiple projects (applies to all icons pretty much)